c_src/ErlUtil.cc

```diff
@@ -1175,6 +1188,23 @@ std::string ErlUtil::formatBinary(unsigned char* buf, size_t size)
     return os.str();
 }
 
+std::string ErlUtil::formatAsString(char* buf, size_t size)
+{
+    return formatAsString((unsigned char*) buf, size);
+}
+
+std::string ErlUtil::formatAsString(unsigned char* buf, size_t size)
+{
+    std::ostringstream os;
+
+    for(unsigned iByte=0; iByte < size; iByte++)
```

Comment:
Since this loop effectively casts each unsigned char to char, this whole function could instead be written as:
```c
    return std:string((char*)buf);
```